src/Hydrators/CarHydrator.php

```diff
@@ -14,4 +14,12 @@ public static function getCars(\PDO $db): array
         $query->setFetchMode(\PDO::FETCH_CLASS, CarEntity::class);
         return $query->fetchAll();
     }
+    public static function getCar(\PDO $db): CarEntity
```

Comment:
this doesnt match the architecture diagram. the ID should be passed into this method as a parameter